yeoman-generator/generators/app/templates/src/plugin.ts.tpl

```diff
@@ -43,7 +43,7 @@ const <%= pluginClass %>: octant.PluginConstructor = class <%= pluginClass %>
   capabilities = {
     supportPrinterConfig: [podGVK],
     supportTab: [podGVK],
-    actionNames: ["action.octant.dev/setNamespace"],
+    actionNames: ["action.octant.dev/setNamespace", "action.octant.dev/setFilter", "action.octant.dev/setContext"],
```

Comment:
These are optional, unless we are going to add examples of creating handlers to react to there actions, I don't think we want to include them in the example plugin template.

This might be a good case for add a new example to the https://github.com/vmware-tanzu/octant-example-plugins repo